runtime/src/accounts_db.rs

```diff
@@ -220,6 +220,16 @@ pub struct ErrorCounters {
     pub invalid_writable_account: usize,
 }
 
+#[derive(Debug, Default, Clone, Copy)]
+pub struct IndexGenerationInfo {}
```

Comment:
why create this?

Reply:
I was trying to set up my next PR to have fewer changes, but I can (and will) move this struct over to that one instead.

Reply:
Done!